api/maven-api-core/src/main/java/org/apache/maven/api/plugin/Mojo.java

```diff
@@ -35,11 +54,12 @@
 @ThreadSafe
 public interface Mojo {
     /**
-     * Perform whatever build-process behavior this {@code Mojo} implements.
-     * This is the main trigger for the {@code Mojo} inside the Maven system,
-     * and allows the {@code Mojo} to communicate errors.
+     * Perform whatever build-process behavior this {@code Mojo} implements. This is
+     * the main trigger for the {@code Mojo} inside the Maven system, and allows the
+     * {@code Mojo} to communicate errors.
      *
-     * @throws MojoException if a problem occurs
+     * @throws Exception if any problem occurs that prevents the mojo from its
+     *                   execution
```

Comment:
I don't think this sentence is correct English.  Either grab the one I pasted below, or rephrase it `if a problem occurs that prevents the mojo from executing` ... ?